applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.hpp

```diff
@@ -594,18 +592,14 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellCrossSection : public Fl
 
         virtual void save(Serializer& rSerializer) const
         {
-            rSerializer.save("T", mThickness);
-            rSerializer.save("L", mLocation);
-            rSerializer.save("O", mOrientationAngle);
+            rSerializer.save("idx", mPlyIndex);
             rSerializer.save("IntP", mIntegrationPoints);
             rSerializer.save("Prop", mpProperties);
         }
 
         virtual void load(Serializer& rSerializer)
         {
-            rSerializer.load("T", mThickness);
-            rSerializer.load("L", mLocation);
-            rSerializer.load("O", mOrientationAngle);
+            rSerializer.load("idx", mPlyIndex);

```

Comment:
i don't understand why all of this should be removed...

Reply:
I only remove storing of the member

It is now accesses through the properties